content/en/docs/setup/install/multicluster/single-network/initial-configuration/index.md

```diff
@@ -0,0 +1,205 @@
+---
+title: Initial Configuration
```

Comment:
Should this be called something like `Install your first primary cluster`? Initial configuration isn't clear.

Reply:
The TOC pointed out that readers might come to the guide with a cluster already installed, or many other different states. Thus, the previous title you mention doesn't work well. `Initial Configuration` works better because these are the steps readers need to take to setup things initially for a multicluster deployment.